context/app/static/js/components/detailPage/BulkDataTransfer/LoggedIn/NoProtectedAccess/NoProtectedAccess.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
+import { StyledContainer } from '../../style';
```

Comment:
If you're traversing farther than a single level, I'd prefer absolute imports using one of the aliases we've defined. `js/components/...`